lib/load.js

```diff
@@ -1,19 +1,36 @@
 const chalk = require('chalk');
-const mock = require('mock-require');
+const cosmiconfig = require('cosmiconfig');
 const resolvePath = require('resolve').sync;
 const webpackLog = require('webpack-log');
 
 const LoadConfigError = require('./LoadConfigError');
 const RequireModuleError = require('./RequireModuleError');
 
+const cwd = process.cwd();
+const { loadJs } = cosmiconfig;
+const prefix = 'webpack';
+const cosmicOptions = {
+  loaders: {
+    '.es6': loadJs,
+    '.flow': loadJs,
+    '.mjs': loadJs,
+    '.ts': loadJs,
+  },
+  searchPlaces: [
+    `${prefix}.config.js`,
+    `${prefix}.config.es6`,
+    `${prefix}.config.flow`,
+    `${prefix}.config.mjs`,
+    `${prefix}.config.ts`,
+    `.${prefix}rc`,
+    'package.json',
+    `.${prefix}rc.json`,
+    `.${prefix}rc.yaml`,
+    `.${prefix}rc.yml`,
+  ],
+};
+
 module.exports = (options = {}) => {
-  const altExtensions = [null, '.es6', '.flow', '.mjs', '.ts'];
```

Comment:
What's this for?

Reply:
well that's been removed. it was part of the old setup, when cosmiconfig didn't have a good options structure for different file extensions.